tests/test_task.py

```diff
@@ -111,3 +201,105 @@ def test_structured_data_classifier_transform_new_data(tmp_dir):
     clf = ak.StructuredDataClassifier(directory=tmp_dir, max_trials=2)
     clf.fit(x_train, y_train, epochs=2, validation_data=(
         x_test, y_test))
+
+
+def test_structured_data_from_csv_regressor(tmp_dir):
+    TRAIN_DATA_URL = "https://storage.googleapis.com/tf-datasets/titanic/train.csv"
```

Comment:
instead of rely on google to download, we would rather put these two files in /tests/resources/titantic/...